File: backend/BakeryStoreApi/DTOs/CheckoutRequest.cs
namespace BakeryStoreApi.DTOs
{
    public class CheckoutRequest
    {
        public int UserId { get; set; }

        public string ReceiverName { get; set; } = string.Empty;

        public string Phone { get; set; } = string.Empty;

        public string Street { get; set; } = string.Empty;

        public string City { get; set; } = string.Empty;

        public string District { get; set; } = string.Empty;

        public string Ward { get; set; } = string.Empty;

        public string PaymentMethod { get; set; } = "Cash";
    }
}

File: app/src/main/java/com/example/bakerystore/models/CheckoutRequest.kt
package com.example.bakerystore.models

import com.google.gson.annotations.SerializedName

data class CheckoutRequest(
    val userId: Int,
    val receiverName: String,
    val phone: String,
    val street: String,
    val city: String,
    val district: String,
    val ward: String,
    val paymentMethod: String,

    @SerializedName("cartItemIds")
    val cartItemIds: List<Int>
)